src/full_node/mempool_manager.py

```diff
@@ -123,8 +123,7 @@ async def add_spendbundle(
         program = best_solution_program(new_spend)
         # npc contains names of the coins removed, puzzle_hashes and their spend conditions
         fail_reason, npc_list, cost = calculate_cost_of_program(
-            program,
-            self.constants.CLVM_COST_RATIO_CONSTANT,
+            program, self.constants.CLVM_COST_RATIO_CONSTANT,
```

Comment:
Not introduced by this change, but up on line 46 - pylint gives the following error:
`src/full_node/mempool_manager.py:46:27: E1136: Value 'SortedDict' is unsubscriptable (unsubscriptable-object)`

```
 pylint --help-msg=unsubscriptable-object
:unsubscriptable-object (E1136): *Value '%s' is unsubscriptable*
  Emitted when a subscripted value doesn't support subscription (i.e. doesn't
  define __getitem__ method or __class_getitem__ for a class). This message
  belongs to the typecheck checker.
```
if this is ok, then add `  # pylint: disable=E1136` at the end of that line

Reply:
OK, i'll fix it